internal/install/command.go

```diff
@@ -73,4 +81,6 @@ func init() {
 	Command.Flags().BoolVarP(&skipIntegrations, "skipIntegrations", "r", false, "skips installation of recommended New Relic integrations")
 	Command.Flags().BoolVarP(&skipLoggingInstall, "skipLoggingInstall", "l", false, "skips installation of New Relic Logging")
 	Command.Flags().BoolVarP(&testMode, "testMode", "t", false, "fakes operations for UX testing")
+	Command.Flags().BoolVar(&verbose, "verbose", false, "increase log level verbosity")
```

Comment:
Consider long flag `debug`, "debug level logging" for description and short flag `v`?

Reply:
`-v` is taken for verbose currently on the root command, which we could change, but `--debug` sounds find to me too.